src/js/index.js

```diff
@@ -78,22 +77,24 @@ var glify = {
       this.maps.push(map);
       map.on('click', function (e) {
         var hit;
+        hit = Shapes.tryClick(e, map); 
```

Comment:
This change will essentially never allow points or lines within a shape to be clicked.

Reply:
Do points  and lines click need to be placed first? I made this change because shapes being clicked would not show my popup box.

Reply:
I can revert this if needed